testapp/src/main/res/layout/settings_menu_activity.xml

```diff
@@ -23,32 +23,27 @@
           android:layout_height="wrap_content"
           android:orientation="vertical">
 
-        <RelativeLayout
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:padding="@dimen/medium_16"
-          android:foreground="?attr/selectableItemBackground">
-
-          <com.google.android.material.switchmaterial.SwitchMaterial
-            android:id="@+id/switchPreviewMode"
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/textInfo"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/lb_preview_mode" />
-        </RelativeLayout>
+            android:layout_margin="@dimen/small_8"
+            android:padding="@dimen/small_8"
+            android:textColor="@android:color/black"
+            tools:text="Build Info" />
 
         <View
-          android:layout_width="match_parent"
-          android:layout_height="@dimen/horizontal_divider_height"
-          android:background="@color/bg_section" />
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/horizontal_divider_height"
+            android:background="@color/bg_section" />
 
-        <androidx.appcompat.widget.AppCompatTextView
-            android:id="@+id/textInfo"
+        <com.google.android.material.switchmaterial.SwitchMaterial
+            android:id="@+id/switchPreviewMode"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_margin="@dimen/small_8"
             android:padding="@dimen/small_8"
-            android:textColor="@android:color/black"
-            tools:text="Build Info" />
+            android:text="@string/lb_preview_mode" />
```

Comment:
I wish we could also add the following lines for this `SwitchMaterial` to highlight the UI.

```
android:textSize="@dimen/text_large_16"
android:textColor="@android:color/black" 
```

Reply:
I prefer our current one.